src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -354,12 +354,12 @@ private void resetCache()
 		}
 
 		final IMEInventory<IAEItemStack> out = this.getInternalHandler();
-
-		if( this.monitor != null )
-		{
-			this.monitor.onTick();
-		}
-
+		/*
+		 * if( this.monitor != null )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)